tests/operation/test_dataflow.py

```diff
@@ -2,11 +2,12 @@
 import json
 import unittest.mock
 
-from dffml.df.types import DataFlow, Input
+from dffml.df.types import DataFlow, Input, Definition
```

Comment:
Let's merge master, I think these changes are showing up because of the unused imports being removed.